### ISSUE TITLE
2_Data/2_DataHandling.ipynb block 82

### ISSUE THREAD

**Issue by yantingMu** [JulianBesems/UD_IntroProject_2022#1]
The second block of _Example with Netflix dataset_ does not run properly on some computers, due to Python3.8 `open()` uses computers' environment to choose "appropriate" encoding, e.g. Windows with North America English will be given the 8bit Windows-1252 character set, while some chars in netflix_titles.csv required UTF-8 (such as in line 8 `70304989, Movie, Automata, Gabe Ibáñez, ...`). 

Thus, we shall write: 
`with open('netflix_titles.csv', encoding='utf-8') as csvfile:`
in line 2, to override the encoding to UTF-8.

It has been Proposed that [this issue should finally get resolved in python 3.15](https://peps.python.org/pep-0686/), but I thought the issue in this repo should be fixed, as this is a project teaching Python 3.8, according to [0_Introduction](https://github.com/JulianBesems/UD_IntroProject_2022/blob/main/0_Introduction/0_Introduction.ipynb).

**Comment by JulianBesems** [comment]
Yes you're right. Thanks for bringing this up. I'll include it in the notebook now.
